test/s3select_test.cpp

```diff
@@ -2347,3 +2347,16 @@ TEST(TestS3selectFunctions, trim11)
 test_single_column_single_row( "select trim(trailing from trim(leading from \"   foobar   \")) from stdin ;" ,"foobar,\n");
 }
 
+TEST(TestS3selectFunctions, likescape)
```

Comment:
need to add multi-row tests, and dynamic-expression tests (not just constant string)